resources/keda-olm-operator.yaml

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
Consider moving this commit to a separate PR, as the changes are unrelated to the release prep.